tobac/merge_split.py

```diff
@@ -7,8 +7,30 @@
     
 """
 
-
-def merge_split_MEST(TRACK, dxy, distance=None, frame_len=5):
+import logging
```

Comment:
@w-k-jones you've done a great job cleaning up the code in general, I appreciate the changes/updates in formatting/text.